### ISSUE TITLE
Fully uppercases names in 'Fix common OCR errors (using OCR replace list)'

### ISSUE THREAD

**Issue by SuperDre** [SubtitleEdit/subtitleedit#5875]
At least with the latest update 3.6.5 it seems to uppercase certain names to all uppercase.

With miss bergens? => BERGENS
Dorman => DORMAN
kimsicle => KIMSICLE
Montaldo => MONTALDO
Mcinerney => MCINERNEY
McInerney => McInerney  (So this is weird, as it doesn't change it)  
o'tooles => O'TOOLES

If you create a new subtitle file and add above as single lines you can easily test it.

I wonder which replace list it's using as I couldn't find any of the words in my local dictionaries

**Comment by cyzs233** [comment]
Are you sure about this? Looks fine on my side.🤔
![hmm](https://user-images.githubusercontent.com/25927091/161383509-a13c5220-01aa-4f9a-8f66-5a1bb991fe65.png)

**Comment by SuperDre** [comment]
This is how it looks on my side:
![image](https://user-images.githubusercontent.com/1668535/161396737-bbf87794-71fa-49fe-9d7e-925fd44a1008.png)

**Comment by SuperDre** [comment]
Hmmm.. now I see something...
![image](https://user-images.githubusercontent.com/1668535/161396940-6296467b-537c-4f65-bba7-2b8cf9485c56.png)
I manually opened _en_names_user.xml_ with notepad++ but I did not see these names in the word list in the application itself.
But why would it use 'en_names_user.xml' for OCR? the file is used for skipping certain names, otherwise it'll be a mess, as it van also contains 'words' that are garbage at the time of skipping. And it seems there is no way to edit this list from within the application.
